### ISSUE TITLE
The picture is upside down in the App

### ISSUE THREAD

**Issue by Inyaheyukan** [mapillary/mapillary_issues#2740]
![screenshot_20171012-160023](https://user-images.githubusercontent.com/4369755/31585942-7734c38a-b1ca-11e7-81e0-e1dba42da59a.png)

When I enter the capture mode, even before starting capture itself, the picture is upside down. 
And later when I capture pitctures, the recorded pictures are also upside down.

### Basic information
* Platform: `Android`
* Version: `7.1.1` 
* System: `bq Aquaris U Plus`

### Steps to reproduce behavior

 1. open the app
 2.  click on capture

### Expected behavior

The sky is up and the street is down.

### Actual behavior

The street is up and the sky is down.

### Additional information

No matter how around I turn the smartphone.
There are no vibrations.

**Comment by katrinhumal** [comment]
Hi @Inyaheyukan and thanks for reporting! @ovivoda will take a look at this. Meanwhile, for any uploads that you've already done that are upside down on Mapillary, you can use this form to mark them as in need of rotation: https://goo.gl/forms/o2nCZrC2KKiezQJz1

**Comment by Inyaheyukan** [comment]
App Version 3.105

**Comment by ovivoda** [comment]
This issue should be fixed in the latest app version.
